### DISCUSSION TITLE
Rebuild on same container

### DISCUSSION THREAD

**Discussion by ghost** [cirruslabs/cirrus-ci-docs#1224] [Q&A] [answered]
Hello there!
I have a question. If I trigger a build and for example, if the build fails, so is it possible to run the build on the same container so that all the data is restored and the build will not start from starting once again ? 
Help Would be appreciated

**Comment by fkorotkov** [comment, marked as answer]
Hey @eartinity,

This is not possible to due nature of Cirrus CI design where all tasks are executed in single use containers or virtual machines.
